src/main/java/com/traveltime/sdk/dto/common/transportation/DrivingTrain.java

```diff
@@ -0,0 +1,28 @@
+package com.traveltime.sdk.dto.common.transportation;
+
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.traveltime.sdk.dto.common.MaxChanges;
+import jakarta.validation.Valid;
+import jakarta.validation.constraints.Positive;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Value;
+import lombok.extern.jackson.Jacksonized;
+
+@Value
+@Builder
+@Jacksonized
+@AllArgsConstructor
+@JsonInclude(JsonInclude.Include.NON_NULL)
+public class DrivingTrain implements Transportation {
+    @Positive(message = "ptChangeDelay must be greater than 0")
+    Integer ptChangeDelay;
+    @Positive(message = "drivingTimeToStation must be greater than 0")
+    Integer drivingTimeToStation;
+    @Positive(message = "parkingTime must be greater than 0")
+    Integer parkingTime;
```

Comment:
Without looking at docs, it's hard to understand how to use these fields. I would suggest to comment these fields, preferably in javaDoc format.

Reply:
I'm not sure that javaDoc is well compatible with Lombok, so I'm planning to move all these params to readme.